tensorrt_llm/_torch/custom_ops/torch_custom_ops.py

```
@@ -1131,3 +1148,536 @@ def record_stream(tensor: torch.Tensor, stream_id: int) -> None:
+    )
+
+
+@torch.library.register_fake("trtllm::cute_dsl_fp8_bmm_blackwell")
+def _(
+    mat_a: torch.Tensor,
+    mat_b: torch.Tensor,
+    input_scale: torch.Tensor,
+    weight_scale: torch.Tensor,
+    out: torch.Tensor,
+) -> None:
+    l, m, k = mat_a.shape[0], mat_a.shape[1], mat_a.shape[2]
```

Ruff (0.12.2)

Ambiguous variable name: `l`

(E741)